autora/cycle/simple.py

```diff
@@ -471,7 +471,9 @@ def _theorist_callback(theorist, data_in: SimpleCycleData):
             data_in.metadata.independent_variables
         )  # The number of independent variables
         x, y = all_observations[:, :n_xs], all_observations[:, n_xs:]
-        new_theorist = copy.copy(theorist)
+        if y.shape[1] == 1:
+            y = y.ravel()
```

Comment:
These lines do not deal with issue #209. 

I was getting an annoying warning message from scikitlearn `LogisticRegression()` when it was expecting a 1D array when fitting the model. It wanted an array with 1-D shape (n,) and not a  single column (n,1). This checks for that and unravels the array. 

Not sure if this could cause other issues when using other models...